source/server-selection/server-selection.rst

```diff
@@ -579,9 +579,20 @@ Passing read preference to mongos and load balancers
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If a server of type Mongos or LoadBalancer is selected for a read operation, the read
-preference is passed to the selected mongos through the use of the
-``SecondaryOk`` wire protocol flag, the ``$readPreference`` query
-modifier or both, according to the following rules.
+preference is passed to the selected mongos through the use of ``$readPreference``
+(as a `Global Command Argument`_ for OP_MSG or a query modifier for OP_QUERY) and, for
+OP_QUERY only, the ``SecondaryOk`` wire protocol flag, according to the following rules.
+
+For OP_MSG:
+```````````
+
+- For mode 'primary', drivers MUST NOT set ``$readPreference``
+
+- For modes 'secondary', 'primaryPreferred', 'secondaryPreferred', and 'nearest', drivers
+  MUST set ``$readPreference``
```

Comment:
If this section is meant to cover all non-primary read preferences, for clarity I would say something like "For all other read preference modes (e.g. secondary, primaryPreferred, ...)" so that it is clear that there aren't any other read preference modes that the driver needs to consider.

Reply:
Updated.